sdk/data/synths.ts

```diff
@@ -160,39 +137,6 @@ export const synths: Record<SynthSymbol, BasicSynth> = {
 		},
 		category: 'forex',
 	},
-	sLINK: {
-		name: 'Chainlink',
-		asset: 'LINK',
-		addresses: {
-			1: '0xbBC455cb4F1B9e4bFC4B73970d360c8f032EfEE6',
```

Comment:
We should keep Chainlink on L1 Ethereum (chainID 1) as it seems to be only deprecated on L2 (chainID 10) so far.